packages/web-react/src/components/Accordion/README.md

```diff
@@ -198,6 +198,12 @@ On top of the API options, the components accept [additional attributes][readme-
 If you need more control over the styling of a component, you can use [style props][readme-style-props]
 and [escape hatches][readme-escape-hatches].
 
+## Dependencies
+
+The Accordion component uses the `Icon` component internally. To ensure correct rendering,
```

Comment:
If it were worded `This component uses the ...` the paragraph could be the same everywhere 😬

Reply:
edited